crates/solvers/openapi.yml

```diff
@@ -568,13 +568,20 @@ components:
             - $ref: "#/components/schemas/Fulfillment"
             - $ref: "#/components/schemas/JitTrade"
         - type: object
+          required:
+            - executedAmount
```

Comment:
I don't think this is required for JIT orders, and assumed to be the full JIT order amount. (I may be remembering wrong though!).

Reply:
Nevermind, it does use this field.